service-workers/service-worker/claim-fetch.https.html

```diff
@@ -32,9 +32,13 @@
 
     // Register a service worker.
     .then(() => service_worker_unregister_and_register(t, script, scope))
-    .then(r => worker = r.installing)
-    .then(() => wait_for_state(t, worker, 'activated'))
+    .then((r) => {
```

Comment:
nit: not sure why you're adding () around r? it seems more consistent with the rest of the file to not add those.

Reply:
Yup, my personal style leaked through while re-writing these callbacks. Should be fixed, now.